### PR TITLE
Make the readme more streamline

### PR DESCRIPTION
**Pull request by fgrunewald**
This new version of the Readme draws more attention to the wiki and migrates all essential information there. This keeps the front page more clean and allows us to fix simple problems in the tutorial/installation instructions without having to edit the Readme every time.

@ricalessandri and @pckroon could you give some general comments feedback?

Also together with this PR the following wiki pages have been created. Ideally someone check those as well. You should be able to edit and modify them as you see fit. Let me know if you have any issues:

https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start
https://github.com/marrink-lab/polyply_1.0/wiki/Installation
https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,82 +3,21 @@
...
-some commonly used macromolecules using different force-fields (martini, gromos) is included.
-In principle the program can be used with any type of force-field (FF).
+Polyply is a python suite designed to facilitate the generation of input files and coordinates for simulating
+(bio)macromolecules such as synthetic polymers or polysacharides. Input files can be generated either from user specified building blocks or by using the polymers availbable in the library. The library currently includes polymer definitions for the GROMOS and Martini force-fields. The [quick start](https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start) section in the wiki gives an overview of the most important commands. In addition some [tutorials][wiki] are provided for more in-depth information on how to use the program. Tutorials for example include, how to generate [Martini polymer systems](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-martini-polymer-melts) or [write input files](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-writing-.ff-input-files). More details on the algorithm and verification can be found in the [publication](https://arxiv.org/abs/2105.05890).
```
Comment (pckroon) [inline README.md:7]: Break over multiple lines, contains at least one typo

File: README.md [hunk 2]
```diff
@@ -3,82 +3,21 @@
...
-Note that at the moment polyply can only generate disordered structures of polymers. All molecules
-that have secondary structure (e.g. DNA, proteins) cannot be generated accurately. At the moment
-chirality is also not taken into account. All polymers are atactic unless a dihedral specifies the 
-chirality.
+## Quick references
+[Installation Guide](https://github.com/marrink-lab/polyply_1.0/wiki/Installation)<br/>
+[FAQs](https://github.com/marrink-lab/polyply_1.0/wiki/FAQs)<br/>
+[Submissions to martini polymer library](https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters)<br/>
+[Tutorial Martini Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-martini-polymer-melts)<br/>
+[Tutorial GROMOS Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-GROMOS-polymer-melts)<br/>
```
Comment (pckroon) [inline README.md:17]: The `<br\>` line breaks look weird in markdown. Change them to `\` (https://stackoverflow.com/questions/26626256/how-to-insert-a-line-break-br-in-markdown)

File: README.md [hunk 3]
```diff
@@ -3,82 +3,21 @@
...
 
 ## Contributions
-The development of polyply is done on [github]. Contributions
-are welcome as [bug reports] and [pull requests]. Note however that the
-decision of whether or not contributions can give authorship on the resulting
-academic paper is left to our sole discretion.
+We are happy to accept submissions of polymer parameters (mostly for the Martini force-field). After a small quality control procedure, parameters are distributed via the Martini library including appropiate citations. To submit parameters simply open an [issue][bug reports]. More details on submitting parameters can be found [here](https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters). The code development of polyply is done on [github]. Contributions are welcome as [bug reports] and [pull requests] from everyone.
```
Comment (pckroon) [inline README.md:20]: Break over multiple lines

**Review by pckroon — CHANGES_REQUESTED** [review]
Couple of nitpicks on the markdown formatting, but the text looks good to me.

File: README.md (hunk 1 above)
Comment (ricalessandri) [reply README.md:7]: typos: polysacharides, availbable

File: README.md (hunk 2 above)
Comment (ricalessandri) [reply README.md:17]: I like the quick references. 

Small thingy: you are capitalizing "Installation Guide" and "Tutorial Martini Polymers" but not "Submissions to martini polymer library" - I would be consistent.

File: README.md (hunk 3 above)
Comment (ricalessandri) [reply README.md:20]: for the rest looks good to me

**Review by ricalessandri — CHANGES_REQUESTED** [review]
Few things noted but overall looks nice

**Comment by ricalessandri** [comment]
I did make also some (cosmetic) changes to:
- https://github.com/marrink-lab/polyply_1.0/wiki/Quick-Start
- https://github.com/marrink-lab/polyply_1.0/wiki/Installation
- https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters

**Comment by fgrunewald** [comment]
@pckroon and @ricalessandri thanks for the quick feedback! I added line breaks and fixed the typos as well as made the quick-reference section consistent. I also added the citation to the archive paper. 

@ricalessandri thanks for the fixes in the wiki sections.

**Review by pckroon — APPROVED** [review]
LGTM!

**Pull request merged**